advanced.txt

```diff
@@ -1186,7 +1186,7 @@ Version im aktuellen Working Tree) sowie `Stashed Changes`
 [IMPORTANT]
 ============
 Sollten Sie versucht haben, einen Stash mit `git stash
-  pop` anzuwenden, wird der Stash 'nicht' automatisch gelöscht.
+  apply` anzuwenden, wird der Stash 'nicht' automatisch gelöscht.
```

Comment:
Das ist schon korrekt so, aber vielleicht missverständlich formuliert. Bei `git stash apply` wird der Stash nie gelöscht. Wenn man aber normalerweise mit `git stash pop` arbeitet, und dann in einen Konflikt gerät, dann hat man Konfliktmarker und löst den Konflikt – der Stash verschwindet aber nicht automatisch. Vermutlich ist es am klarsten, wenn Du in den Satz “im Konfliktfall” einbaust.